### ISSUE TITLE
Unable to add items 

### ISSUE THREAD

**Issue by mark-prins** [openmsupply/mobile#1559]
## Describe the bug

Unable to add items from a master list when creating a customer invoice.
Raised by Kevin - Côte d'Ivoire "Un fenêtre de chargement apparait et disparait sans information"

### To reproduce

Steps to reproduce the behaviour:

1. Create a new customer invoice
2. Click on Add Master List Items

### Expected behaviour

Should show a selection of master lists? Or not have the button if none available?

### Version and device info

- App version: 3.0.0
- Tablet model: Emulated. Reported by client on Samsungs

**Comment by joshxg** [comment]
Translation for those who don't speak french 

```
A loading window appears and disappears without information
```

This is because there are no master lists for that customer. We are working on a feature for master list selection #1551 and there is discussion here on dealing with the situation where the customer has no master lists here: https://github.com/openmsupply/mobile/pull/1551#discussion_r348097621 

This has been how the button has worked forever but hoping to get it into the next version (3.1, ideally next week)

**Comment by mark-prins** [comment]
ah. thanks Josh. I'd missed that issue

**Comment by mark-prins** [comment]
ah. thanks Josh. I'd missed that issue

**Comment by mark-prins** [comment]
![add-master-list-items-to-ci](https://user-images.githubusercontent.com/9192912/69292280-90956d00-0c6a-11ea-962c-9a7e4c925acf.gif)

Forgot to upload this

**Comment by joshxg** [comment]
Should be fixed with #1575
